src/wizards/debugging/startDebugging.ts

```diff
@@ -48,3 +54,16 @@ export async function startDebugging(): Promise<void> {
 
   await debug.startDebugging(workspaceFolder, debugConfig);
 }
+
+const DEPLOY_TO_OPENSHIFT_ADVERTISEMENT_MEMENTO = 'deployToOpenshift.advertisementShown';
+
+function hasShownDeployToOpenShiftPopUp(): boolean {
+  return QuarkusContext.getExtensionContext().globalState.get(DEPLOY_TO_OPENSHIFT_ADVERTISEMENT_MEMENTO, false);
+}
+
+function showDeployToOpenShiftPopUp(): void {
+  const DEPLOY_TO_OPENSHIFT_ADVERTISEMENT = 'You can deploy to OpenShift right from VS Code using ' + //
+      '"Quarkus: Deploy Current Project ot OpenShift" from the command palette';
```

Comment:
Update to match based on https://github.com/redhat-developer/vscode-quarkus/pull/374 .